### DISCUSSION TITLE
v1.9.2

### DISCUSSION THREAD

**Discussion by olljanat** [burmilla/os#107] [General]
# About this release
BurmillaOS is community maintained RancherOS compatible operating system.

**Versioning**
1.9.x versions will stay on 4.14.x kernel versions and are backward compatible with RancherOS 1.5.x versions.
2.0.x versions will contain bigger changes like 5.10.x kernel, etc.

**Upgrade**
https://burmillaos.org/docs/installation/upgrading/

# Release notes
- Linux 4.14.241
- Buildroot: 2021.02.3
- Docker 20.10.8 by default
- Added nvi to base console docker image. #103
- Bump up docker-compose 1.29.2

<hr /><em>This discussion was created from the release <a href='https://github.com/burmilla/os/releases/tag/v1.9.2'>v1.9.2</a>.</em>

**Comment by prologic** [comment]
Could you just drop a few lines or a link to how to upgrade existing BurmillaOS nodes? I _assume_ we can upgrade in place with a `ros ...` command?

**Reply by MattKobayashi** [reply]
You'll be able to upgrade with the command `sudo ros os upgrade` once [this PR](https://github.com/burmilla/releases/pull/1) is merged.

**Reply by prologic** [reply]
Nice 👌 thanks! 🙇‍♂️

**Reply by olljanat** [reply]
Ah. I forgot to add it to https://github.com/burmilla/releases/blob/v1.9.x/releases.yml

Now `sudo ros os list` should show new version and upgrade works with `sudo ros os upgrade` like it is on guidance.

**Reply by olljanat** [reply]
I quickly tested that at least v1.9.1 -> v1.9.2 upgrade worked fine on my env.

**Reply by prologic** [reply]
I will upgrade soon then 👌

**Comment by thorstenthomsen** [comment]
I can't find AMI's in Amazon cloud. When will they be available?
